tests/webTables.spec.ts

```diff
@@ -0,0 +1,515 @@
+import { test, expect } from '@playwright/test';
+import { spec } from 'node:test/reporters';
+
+const timeout = 7000
+
+test.beforeEach(async ({ page }) => {
+    await page.goto('/')
+
+});
+
+test('Validate the pet name city of the owner', async ({ page }) => {
+    /* 1. Select the OWNERS menu item in the navigation bar and then select "Search" from the drop-down menu */
+    // Navigate to the owners table
+    await page.getByText('Owners').click()
+    await page.getByText('Search').click()
+
+    /* 2. In the list of Owners, locate the owner by the name "Jeff Black". 
+    Add the assertions that this owner is from the city of "Monona" and he has a pet with a name "Lucky" */
+    // Find Jeff Black in the table by searching
+    await page.getByRole('textbox').fill("black")
+    await page.getByText('Find Owner').click()
+
+    const row = page.getByRole('row').locator('td')
```

Comment:
Shamelessly.  I was trying to formulate that question, but I didn't quite get it.  I re-watched the videos and realized I need to take physical notes rather than purely active listening.